website/pages/component/select/guidelines.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ComingSoon from 'components/ComingSoon';
+
+export default function Guidelines() {
```

Comment:
Does next automatically add `displayName`? if not it would be helpful to have these. I believe there's a babel plugin that will automatically add them.

Reply:
I tried adding the babel plugin and it didn't work, but I can be sure to add in displayNames for each component myself.